deploy.sh

```diff
@@ -4,30 +4,30 @@ set -eo pipefail
 # Check for release branch - not using grep as set -e means it fails script
 RELEASE_BRANCH=$(echo "$TRAVIS_BRANCH" |  sed -n 's/^release\-/&/p')
 
-#Get the release type (dev/master) from the branch name
+# Get the release type (dev/master) from the branch name
 TYPE="$TRAVIS_BRANCH"
 
-if ([ "$TRAVIS_PULL_REQUEST" == "false" ]  && [ "${TRAVIS_REPO_SLUG}" == "ScottLogic/StockFlux" ] && ([ "$TYPE" == "dev" ] || [ "$TYPE" == "master" ] || [ -n "$RELEASE_BRANCH" ]))
+if ([ "$TRAVIS_PULL_REQUEST" == "false" ]  && [ "${TRAVIS_REPO_SLUG}" == "rjmcneill/StockFlux" ] && ([ "$TYPE" == "chore/deployable-build" ] || [ "$TYPE" == "dev" ] || [ "$TYPE" == "master" ] || [ -n "$RELEASE_BRANCH" ]))
 then
-    #Clone the latest gh-pages
-    git clone https://github.com/ScottLogic/StockFlux.git --branch gh-pages gh-pages
+    # Clone the latest gh-pages
+    git clone https://github.com/rjmcneill/StockFlux.git --branch gh-pages gh-pages
 
-    #Get line with version from the file -> get the second word -> remove quotes around the value
-    VERSION=$(grep "version" package.json | awk -v N=$2 '{print $2}' | cut -d \" -f2)
+    # Get line with version from the file -> get the second word -> remove quotes around the value
+    VERSION=$(grep -m 1 "version" package.json | awk -v N=$2 '{print $2}' | cut -d \" -f2)
```

Comment:
Would `grep "\"version\":"` work too? Just thinking what might be more robust (i.e. what would happen if the order of the keys in package.json changed).